vision/google/cloud/vision/_gax.py

```diff
@@ -30,24 +30,25 @@ def __init__(self, client=None):
         self._client = client
         self._annotator_client = image_annotator_client.ImageAnnotatorClient()
 
-    def annotate(self, image, features):
+    def annotate(self, images):
```

Comment:
1. Sure.
2. Not sure how this is a breaking change. A user should not call `annotate` directly. Usage of `Image.detect()` has not changed.